Core/Core/CourseSync/CourseSyncSelector/Model/Entities/CourseSyncSelectorEntry.swift

```diff
@@ -108,3 +122,25 @@ struct CourseSyncSelectorEntry {
         files[index].state = state
     }
 }
+
+extension CourseSyncSelectorEntry.File {
+    static func make(
+        id: String,
+        displayName: String,
+        fileName: String = "File",
+        url: URL = URL(string: "1")!,
+        mimeClass: String = "jpg",
+        state: CourseSyncSelectorEntry.State = .loading(nil),
+        selectionState: ListCellView.SelectionState = .deselected
+    ) -> CourseSyncSelectorEntry.File {
+        .init(
+            id: id,
+            displayName: displayName,
+            fileName: fileName,
+            url: url,
+            mimeClass: mimeClass,
+            state: state,
+            selectionState: selectionState
+        )
+    }
+}
```

Comment:
Let's wrap this into an `#if DEBUG` statement.
```suggestion
#if DEBUG

extension CourseSyncSelectorEntry.File {
    static func make(
        id: String,
        displayName: String,
        fileName: String = "File",
        url: URL = URL(string: "1")!,
        mimeClass: String = "jpg",
        state: CourseSyncSelectorEntry.State = .loading(nil),
        selectionState: ListCellView.SelectionState = .deselected
    ) -> CourseSyncSelectorEntry.File {
        .init(
            id: id,
            displayName: displayName,
            fileName: fileName,
            url: url,
            mimeClass: mimeClass,
            state: state,
            selectionState: selectionState
        )
    }
}

#endif
```